### PR TITLE
Remove regex lookbehinds to support Safari 16.3 or older 

### PR DESCRIPTION
**Pull request by myzkyy**
Lookbehind assertion is not supported by Safari 16.3 (Released on Jan 23, 2023) or older. It causes errors: `SyntaxError: Invalid regular expression: invalid group specifier name.`

cf. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Lookbehind_assertion#browser_compatibility

This PR removes all of them.

### CODE REVIEW

**Comment by poteboy** [comment]
@kotarella1110  @taishinaritomi 
Hey, I want to hear your opinion on the approach this PR took. This kinda led me to revisit the core purpose of this regex:
- Most libraries let bundler plugins handle CSS minification.
- We create some CSS at runtime (Therefore we need some minification logic like this regex).
- But is it our job to also minify this? Especially when tools like [matthiasmullie/minify](https://github.com/matthiasmullie/minify) exist.
- On the other hand, keeping external dependencies low is good for bundle size and maintenance.

It's a balance. Should we keep this minify process or let other tools handle it?

**Comment by myzkyy** [comment]
@poteboy Thank you for your consideration. I wonder what the meaning of runtime minification is. Minification is for traffic efficiency, isn't it?

By the way, as you may already know, this error blocks running scripts on the website. For example, In https://www.kuma-ui.com/ users can't open the hamburger menu if they use Safari 16.3. I insist this is a critical problem for production use.

**Comment by kotarella1110** [comment]
If it's a reduction logic of the extent to remove spaces, I believe it's fine to keep it. If you want to further reduce it, the user should use a minify tool on their side.

File: packages/sheet/src/sheet.ts [hunk 1]
```diff
@@ -74,11 +74,11 @@ export class Sheet {
...
     breakpoint: string
   ): void {
-    css = css.replace(cssPropertyRegex, "");
+    css = css.replace(cssPropertyRegex, "$1$2");
     const mediaCss =
       `@media (min-width: ${breakpoint}) { .${className} { ${css} } }`.replace(
         removeSpacesExceptInPropertiesRegex,
-        ""
+        "$1$2$3"
       );
```
Comment (kotarella1110) [inline packages/sheet/src/sheet.ts:82]: @myzkyy Is it possible to extract this whitespace removal logic as a function?

example:

```ts
const removeSpacesExceptInProperties = (str: string) => str.replace(removeSpacesExceptInPropertiesRegex, "$1$2$3")
```

Comment (myzkyy) [reply packages/sheet/src/sheet.ts]: @kotarella1110 Done.

**Comment by poteboy** [comment]
duplicate changesets found

**Comment by myzkyy** [comment]
> duplicate changesets found

Merged.

**Review by kotarella1110 — APPROVED** [review]
Thanks! LGTM 🎉

File: packages/sheet/src/sheet.ts [hunk 2]
```diff
@@ -65,32 +68,28 @@ export class Sheet {
...
   private _addPseudoRule(
     className: string,
     pseudo: SystemStyle["pseudo"][number]
   ) {
-    const css = pseudo.base.replace(cssPropertyRegex, "");
-    const pseudoCss = `.${className}${pseudo.key} { ${css} }`.replace(
-      removeSpacesExceptInPropertiesRegex,
-      ""
+    const pseudoCss = removeSpacesExceptInProperties(
+      `.${className}${pseudo.key} { ${pseudo.base} }`
```
Comment (myzkyy) [inline packages/sheet/src/sheet.ts:92]: I noticed unintended behavior change. I'll fix here.

Comment (myzkyy) [reply packages/sheet/src/sheet.ts:92]: `removeSpacesExceptInProperties` contains all of `removeSpacesAroundCssPropertyValues` regexp, so this diff causes no behavior change actually.

Comment (myzkyy) [reply packages/sheet/src/sheet.ts:92]: Anyway I added commit c34ace78b0dc028299f022c6d9c5eb41d10d377c

Comment (myzkyy) [reply packages/sheet/src/sheet.ts]: @kotarella1110 Could you review change c34ace78b0dc028299f022c6d9c5eb41d10d377c?
I added some refactoring, too.

Comment (kotarella1110) [reply packages/sheet/src/sheet.ts]: thanks! will review 👍

File: .changeset/mighty-coats-breathe.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@kuma-ui/sheet": minor
```
Comment (kotarella1110) [inline .changeset/mighty-coats-breathe.md:2]: @myzkyy I think the patch version is better because this is a bugfix.

Comment (myzkyy) [reply .changeset/mighty-coats-breathe.md]: @kotarella1110 Fixed 477c3ffa64ea19d7ad63e94e2eeb8bad12e18b82

**Review by kotarella1110 — APPROVED** [review]
looks good 👍

**Pull request merged**